### ISSUE TITLE
Convert longitudinal FS to new I/O interface library

### ISSUE THREAD

**Issue by glasserm** [Washington-University/HCPpipelines#255]
Hi Jure,

Would you be willing to convert this code to the new I/O interface library that Tim Coalson wrote?  It is much simpler to use than the style this is currently in (which comes from Tim Brown)?

https://github.com/Washington-University/HCPpipelines/blob/94d31f411d2552b775776a24ad14bedfbec9b9ce/FreeSurfer/LongitudinalFreeSurferPipeline.sh

Thanks,

Matt.

**Comment by demsarjure** [comment]
Yeah, sure. Can you just point me to a recent use of the new interface, so I do not use something outdated again. Thanks!

**Comment by glasserm** [comment]
The new PreFreeSurfer is an example.  If you like, you could wait till Yash converts FreeSurfer for a very similar example.

**Comment by demsarjure** [comment]
I will wait for FreeSurfer since it is much more similar and will be easier to use for adaptation.

**Comment by coalsont** [comment]
You can also look at the simple example at the top of `newopts.shlib`:

https://github.com/Washington-University/HCPpipelines/blob/master/global/scripts/newopts.shlib

This section is meant to be copypasted into a new file as a starting point for any new script in the pipelines.  Fill out the arguments section, and write the processing code below the whole thing.
